lib/features/composer/presentation/extensions/file_upload_extension.dart

```diff
@@ -23,22 +20,17 @@ extension FileUploadExtension on FileUpload {
     return base64;
   }
 
-  Future<FileInfo?> toFileInfo() async {
+  FileInfo? toFileInfo() {
     if (base64Data != null) {
-      final bytes = await compute(convertBase64ToBytes, base64Data!);
```

Comment:
is it ok in performance aspect?

Reply:
Still ok, because `compute` only improves performance on mobile, not on web.

Reply:
so why not keep it for mobile?

Reply:
1. We haven't used it anywhere
2. FileUpload is an object of the `html_editor_enhanced` library. We only use `html_editor_enhanced` for web. So we can't write `compute` in the extension of `FileUpload`
3. If we want to keep that code, we should copy it to another `Utils` class,